source/terms/information_elements/mids_information_elements_draft.csv

```diff
@@ -1,29 +1,29 @@
+mids:,Name,Name,"A name given to the object. Any string of characters and/or numbers by which the object is referenced within a collection. For example, the name the specimen is stored under, its scientific or taxonomic name if known, how it is labelled, etc. This name is not necessarily its name according to an accepted scientific classification, identification, or taxonomic determination (i.e., scientific name) but it often can be the same as that.",Information to aid the discoverability of specimens by users and the ability to retrieve them within a collection.,"It is recommended to follow standard nomenclators, e.g. the International Plant Names Index (IPNI), MycoBank, AlgaeBase, Mindat, etc","Colias croceus (Geoffroy, 1785) | Ophthalmosaurus | Australian Baobab | Scottish rock",http://www.w3.org/2000/01/rdf-schema#Class,Yes (all),No,original,2024/05/16,
+mids:,SpecimenType,Specimen Type,"High-level term to delimit and define specimens. For example: preserved specimen, fossil specimen, as opposed to observation.",To delimit the specimens to which the MIDS specification refers.,It is recommended to use the ontology being developed for the DwC term MaterialEntity.,PreservedSpecimen | FossilSpecimen,http://www.w3.org/2000/01/rdf-schema#Class,Yes (all),No,original,2024/05/16,
+mids:,ObjectType,Object Type,General term to describe the kind of specimen. In combination with SpecimenType - hierarchical; a more specific classification than described by SpecimenType.,"To enable curators to determine equipment/method/cost of imaging the specimen, to enable researchers to determine equipment/method required for viewing/analysing the specimen, to enable curators/researchers to know in which collection/location within the institution the specimen is held.",It is recommended to use the ontology being developed for the DwC term Preparations.,Microscope slide | pinned insect | herbarium sheet,http://www.w3.org/2000/01/rdf-schema#Class,Yes (all),No,original,2024/05/16,
+mids:,License,License,License under which the specimen data are published,To enable all users to determine how they can use the specimen data,Use Creative Commons license framework. Use as unrestricted license as possible.,CC0 | CC BY | CC BY-SA,http://www.w3.org/2000/01/rdf-schema#Class,Yes (all),Yes,original,2024/05/16,
+mids:,Modified,Modified,"UTC date/time of date/time of first creation or subsequent modification, if any",To enable users (humans/machines) to determine if there have been any changes to the record since it was last viewed.,"Use ISO 8601. UTC date/time, extended format (for human readability)",2021-01-20 10:54:18,http://www.w3.org/2000/01/rdf-schema#Class,Yes (all),No,original,2024/05/16,

```

Comment:
```suggestion
mids:,Modified,Modified,"UTC date/time of date/time of first creation or subsequent modification, if any",To enable users (humans/machines) to determine if there have been any changes to the record since it was last viewed.,"Use ISO 8601. UTC date/time, extended format (for human readability)",2021-01-20T10:54:18,http://www.w3.org/2000/01/rdf-schema#Class,Yes (all),No,original,2024/05/16,

```

Should we include more examples without a time? With a timezone?